css/dgrid.css

```diff
@@ -124,6 +124,10 @@ html.has-mozilla .dgrid-focus {
   position: absolute;
   top: -9999px;
 }
+.dgrid-scrollbar-measure-child {
+  width: 10px;
+  height: 10px;
+}
```

Comment:
This needs to be modified in the `styl` and regenerated from there :)

Though when we backport to 0.4 it'll need to be in the CSS anyway.